_plugins/regex.rb

```diff
@@ -0,0 +1,12 @@
+module Jekyll
+  module RegexFilter
+    def replace_regex(input, reg_str, repl_str)
+      re = Regexp.new reg_str
+
+      # This will be returned
+      input.gsub re, repl_str
```

Comment:
Probably more readable to do `return input.gsub re, repl_str` than to put a comment pointing out Ruby's implicit return :-)

Reply:
Fixed!